index.php

```diff
@@ -466,6 +466,7 @@
 
 	$content = mb_convert_encoding($a->page["content"], 'HTML-ENTITIES', "UTF-8");
 
+	/// @TODO one day, kill those error-surpressing @ stuff, or PHP should ban it
```

Comment:
Yes, we don't write PHP itself here. :-) but at least the `@` can be removed and instead some more securing code being added, like `file_exists()` or `is_readable()` should be called before a not-always available include file is being loaded.